plugin.video.mlbtv/resources/lib/globals.py

```diff
@@ -0,0 +1,591 @@
+
+
+def getFavTeamId():
+    #possibly use the xml file in the future
+    #http://mlb.mlb.com/shared/properties/mlb_properties.xml  
+    team_ids = {'Arizona Diamondbacks':'109',
+                'Atlanta Braves':'144',
+                'Baltimore Orioles':'110',
+                'Boston Red Sox':'111',
+                'Chicago Cubs':'112',
+                'Chicago White Sox':'145',
+                'Cincinnati Reds':'113',
+                'Cleveland Indians':'114',
+                'Colorado Rockies':'115',
+                'Detroit Tigers':'116',
+                'Houston Astros':'117',
+                'Kansas City Royals':'118',
+                'Los Angeles Angels':'108',
+                'Los Angeles Dodgers':'119',
+                'Miami Marlins':'146',
+                'Milwaukee Brewers':'158',
+                'Minnesota Twins':'142',
+                'New York Mets':'121',
+                'New York Yankees':'147',
+                'Oakland Athletics':'133',
+                'Philadelphia Phillies':'143',
+                'Pittsburgh Pirates':'134',
+                'St. Louis Cardinals':'138',
+                'San Diego Padres':'135',
+                'San Francisco Giants':'137',
+                'Seattle Mariners':'136',
+                'Tampa Bay Rays':'139',
+                'Texas Rangers':'140',
+                'Toronto Blue Jays':'141',
+                'Washington Nationals':'120'}
+
+    print FAV_TEAM
+    fav_team_id = team_ids[FAV_TEAM]
+    print fav_team_id
+    #except:
+    #pass
+
+    return  fav_team_id
+
+def getAudioVideoInfo():
+    #SD (800 kbps)|SD (1600 kbps)|HD (3000 kbps)|HD (5000 kbps)
+    if QUALITY == 'SD (800 kbps)':        
+        video_info = { 'codec': 'h264', 'width' : 512, 'height' : 288, 'aspect' : 1.78 }        
+    elif QUALITY == 'SD (1200 kbps)':
+        video_info = { 'codec': 'h264', 'width' : 640, 'height' : 360, 'aspect' : 1.78 }        
+    else:
+        #elif QUALITY == 'HD (2500 kbps)' or QUALITY == 'HD (3500 kbps)' or QUALITY == 'HD (5000 kbps)':
+        video_info = { 'codec': 'h264', 'width' : 1280, 'height' : 720, 'aspect' : 1.78 }        
+
+    audio_info = { 'codec': 'aac', 'language': 'en', 'channels': 2 }
+    return audio_info, video_info
+
+def getConfigFile():
+    '''
+    GET http://lwsa.mlb.com/partner-config/config?company=sony-tri&type=nhl&productYear=2015&model=PS4&firmware=default&app_version=1_0 HTTP/1.0
+    Host: lwsa.mlb.com
+    User-Agent: PS4Application libhttp/1.000 (PS4) libhttp/3.15 (PlayStation 4)
+    Connection: close
+    '''
+    url = 'http://lwsa.mlb.com/partner-config/config?company=sony-tri&type=nhl&productYear=2015&model=PS4&firmware=default&app_version=1_0'
+    req = urllib2.Request(url)       
+    req.add_header("Connection", "close")
+    req.add_header("User-Agent", UA_PS4)
+
+    response = urllib2.urlopen(req, '')
+    json_source = json.load(response)   
+    response.close()
+    
+
+def setViewMode():
```

Comment:
Agreed. I think I put this in there because the addon was reverting back to a different view type, but it very well could have been something I was doing. I will remove the viewmode code.